scripts/make_port_status.py

```diff
@@ -113,12 +113,21 @@ def get_mathlib4_module_commit_info(contents):
 prs = {}
 fetch_args = ['git', 'fetch', 'origin']
 nums = []
+
+sync_prs = dict()
+labels = dict()
+
 mathlib4repo = github.Github(github_token).get_repo("leanprover-community/mathlib4")
 for pr in mathlib4repo.get_pulls(state='open'):
     if pr.created_at < datetime.datetime(2022, 12, 1, 0, 0, 0):
         continue
     if 'no-source-header' in (l.name for l in pr.labels):
         continue
+    if 'mathlib3-pair' in (l.name for l in pr.labels):
+        for file in (f.filename for f in pr.get_files()):
+            sync_prs[file] = sync_prs.get(file, set()).union([pr.number])
+    if 'mathlib-port' in (l.name for l in pr.labels):
```

Comment:
We have a bunch of PRs for which this label is missing; I think it would be better to preserve the old semantics where we detect ports based on the files they touch, and not the presence of the label.

Reply:
while the existing `prs` dict (few lines further down) seems not to be used otherwise I guess I can just use that one.